src/urllib3/response.py

```diff
@@ -1,9 +1,11 @@
 import io
+import json
```

Comment:
Let's alias this import too so we don't have a name conflict with `HTTPResponse.json()`